src/commcare_cloud/ansible/deploy_postgres.yml

```diff
@@ -15,7 +15,7 @@
     - {role: postgresql, tags: 'postgresql'}
 
 - name: pgbouncer
-  hosts: postgresql:pg_standby
+  hosts: postgresql:pg_standby!citusdb
```

Comment:
does this mean that `citusdb` skips pgbouncer? How does this get pgbouncer onto the workers?

Reply:
It get's handled in `deploy_citusdb.yml`